src/client/app/translations/data.ts

```diff
@@ -128,20 +128,16 @@ const LocaleTranslationData = {
 		"default.file.size.limit": "Default File Size Limit",
 		"default.help.url": "Documentation URL",
 		"default.time.zone": "Default Time Zone",
-		"default.meter.minimum.value": "Default meter minimum reading value check",
-		"default.meter.maximum.value": "Default meter maximum reading value check",
-		"default.meter.minimum.date": "Default meter minimum reading date check",
-		"default.meter.maximum.date": "Default meter maximum reading date check",
```

Comment:
we don't need to edit date, just edit the maximum and minimum value like
"default.minimum.value": "Default minimum reading value check",
"default.maximum.value": "Default maximum reading value check"
and do the same for translations in different languages with a bulb icon.
Also when you do these changes, the previous instances of these keys must also be changed